### DISCUSSION TITLE
v4.5.1

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5358] [Announcements]
v4.5.1

Code v1.68.1

Upgrading is as easy as installing the new version over the old one. code-server
maintains all user data in `~/.local/share/code-server` so that it is preserved in between
installations.

### Changed

- We now use `release/v<0.0.0>` for the release branch name so it doesn't
  conflict with the tag name
- Added `.prettierignore` to ignore formatting files in `lib/vscode`

### Added

- Allow more comprehensive affinity config in Helm chart
- Added custom message in Homebrew PR to make sure code-server maintainers are
  tagged
- Allow setting `priorityClassName` via Helm chart
- Added troubleshooting docs to `CONTRIBUTING.md`

### Fixed

- Removed default memory limit which was set via `NODE_OPTIONS`
- Changed output in pipe to make it easier to debug code-server when doing live
  edits
- Fixed display-language patch to use correct path which broke in 4.5.0
- Fixed multiple code-server windows opening when using the code-server CLI in
  the Integrated Terminal
- Fixed Integrated Terminal not working when web base was not the root path

### Security

- Updated `glob-parent` version in dependencies

## New Contributors
* @bbbboom made their first contribution in https://github.com/coder/code-server/pull/5317
* @timbrd made their first contribution in https://github.com/coder/code-server/pull/5255

**Full Changelog**: https://github.com/coder/code-server/compare/v4.5.0...v4.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/coder/code-server/releases/tag/v4.5.1'>v4.5.1</a>.</em>

**Comment by Coachflux** [comment]
./code-server is not working on Android phone please is there any other way to run this

**Reply by lg-nineoneninethreenine** [reply]
It's due to one of the dependencies named Argon2, which hashes our password. Seems like it doesn't go well with ARM, but @jsjoeio delivered a fix, I think last year, or before March this year. Iys weird seeing it happen again.

**Reply by Coachflux** [reply]
Please send me a guide on how to use ![Screenshot_20220810-140358_1.png](https://user-images.githubusercontent.com/110670400/183909521-a9fd5953-41be-4172-8271-0da32de95fd4.png)

How can I open a folder and also clone repository. Please I really need your help with it.

**Reply by lg-nineoneninethreenine** [reply]
Oi, you are seeing source control! Open the files part, and then open a folder or clone a repo.

**Reply by Coachflux** [reply]
I'm really sorry for disturbing you my dear, but I have tried clone repository, it's not working. I also read the read doc, I followed it all but still not working. Please is their any other to download git on android 32bit![Screenshot_20220810-182812.png](https://user-images.githubusercontent.com/110670400/183979004-b7a69879-ef68-47c6-a31f-79eed11ca1c8.png)

**Reply by lg-nineoneninethreenine** [reply]
You are running this on Android?

**Comment by Coachflux** [comment]
Yes, please guild me through this

**Reply by lg-nineoneninethreenine** [reply]
Can you first tell how you are running code-server on Android? I want to know.

**Reply by Coachflux** [reply]
I just learnt it from You tube but they didn't give more details on clone repository.
